src/items/equipment/valai_band.json

```diff
@@ -0,0 +1,79 @@
+{
+  "_id": "scXy9beHrV8GdtwX",
+  "name": "Valai Band",
+  "type": "technological",
+  "_stats": {
+    "coreVersion": "10.288",
+    "createdTime": 1669870950847,
+    "lastModifiedBy": "SPRszH5sDdJlGxJn",
+    "modifiedTime": 1669871254547,
+    "systemId": "sfrpg",
+    "systemVersion": "0.21.0"
+  },
+  "effects": [],
+  "img": "icons/svg/mystery-man.svg",
+  "system": {
+    "type": "",
+    "abilityMods": {
+      "parts": []
+    },
+    "ammunitionType": "",
+    "attributes": {
+      "ac": {
+        "value": ""
+      },
+      "customBuilt": false,
+      "dex": {
+        "mod": ""
+      },
+      "hardness": {
+        "value": ""
+      },
+      "hp": {
+        "max": "",
+        "value": 7
+      },
+      "size": "medium",
+      "sturdy": false
+    },
+    "attuned": false,
+    "bulk": "L",
+    "capacity": {
+      "max": 15,
+      "value": 15
+    },
+    "container": {
+      "contents": [],
+      "includeContentsInWealthCalculation": true,
+      "isOpen": true,
+      "storage": []
+    },
+    "critical": {
+      "parts": []
+    },
+    "damage": {
+      "parts": []
+    },
+    "description": {
+      "chat": "",
+      "gmnotes": "",
+      "short": "",
+      "unidentified": "",
+      "value": "<p>Developed by the Circle of Journeys to provide safer travel for young @UUID[Compendium.sfrpg.races.6qGAoeERMRvuQjUw]{vlakas} during valai season, this metal band is traditionally worn around the neck in an echo of the vlakan Leader’s Band, but can also be worn on an arm, leg, or similar appendage. When activated, it provides environmental protection within a 10-foot radius of the wearer that is equivalent to wearing armor (Core Rulebook 196). It may also, at the wearer’s discretion, emit a homing signal every 10 seconds for 24 hours at the cost of additional charge. The homing signal can be picked up by most devices capable of receiving signals and tracked back to the source with a successful DC 15 @UUID[Compendium.sfrpg.rules.GMkLZsN3a7YPvA03.JournalEntryPage.76u4HnGcAlySicb4]{Engineering} or @UUID[Compendium.sfrpg.rules.GMkLZsN3a7YPvA03.JournalEntryPage.UDTsEIldhXTn0VJA]{Survival} check. A single valai band’s homing signal has a range of 100 miles over flat terrain (halved in forests or hills and quartered in mountainous terrain or underwater). If multiple valai bands are present within the same 10-foot radius, the homing range increases by a factor of 2 for each additional band, to a maximum of a planetary range.</p><p><strong>Usage</strong> 1/8 Hours</p>"
```

Comment:
What's with the Usage entry in the description?

Reply:
Usage is 1 every 8 hours. we do not have that as an option.

Reply:
Had to add 8 hours to the UsagePer options to complete this one.